api/client/builder/types.go

```diff
@@ -346,6 +346,70 @@ func (p *ExecutionPayload) ToProto() (*v1.ExecutionPayload, error) {
 	}, nil
 }
 
+func FromProto(payload *v1.ExecutionPayload) (ExecutionPayload, error) {
```

Comment:
Helper methods to convert the payload to a builder api comformant type

Reply:
typo in description :smile: